plugins/modules/gitlab_runner.py

```diff
@@ -37,10 +37,17 @@
   - community.general.gitlab
 
 options:
+  group:
+    description:
+      - ID or full path of the group in the form group/subgroup.
+      - Mutually exclusive with I(owned) and I(project).
+    type: str
+    version_added: '6.2.0'
   project:
     description:
       - ID or full path of the project in the form of group/name.
       - Mutually exclusive with I(owned) since community.general 4.5.0.
+      - Mutually exclusive with I(group) since community.general 6.2.0.
```

Comment:
Since it is mutually exclusive with `group` since `group` has been added (namely in this PR), the version doesn't need to be mentioned here:
```suggestion
      - Mutually exclusive with I(group).
```
(Same below.)

In the case of `owned`, the `owned` option was added in c.g 2.0.0, but it was only mutually exclusive with `project` from c.g 4.5.0 on. That's why the version is there in the previous entry.